src/sourmash/cli/compare.py

```diff
@@ -82,6 +80,18 @@ def subparser(subparsers):
     subparser.add_argument(
         '-p', '--processes', metavar='N', type=int, default=None,
         help='Number of processes to use to calculate similarity')
+    subparser.add_argument(
+        '--distance-matrix', action='store_true',
+        help='output a distance matrix, instead of a similarity matrix'
+    )
+    subparser.add_argument(
+        '--similarity-matrix', action='store_false',
```

Comment:
this one isn't documented `command-line.md`. Is that ok?

Reply:
I think so - it shows up in `--help`, and it's the default, so hopefully it's obvious.

(the command-line docs aren't comprehensive; they merely try to be useful :)